shared/common-adapters/section-list.desktop.tsx

```diff
@@ -279,21 +279,15 @@ class SectionList<T extends Section<any, any>> extends React.Component<Props<T>,
           onScroll={this.onScroll}
         >
           {renderElementOrComponentOrNot(this.props.ListHeaderComponent)}
-          <DelayedMounting delay={0}>
-            {/* The delayed mounting is needed for the list to render
-              correctly. Otherwise if the first `sections` prop passed into
-                SectionList is non-zero, the offset is wrong causing first
-            item in the first section to be hidden.*/}
```

Comment:
This is the bug. In InfoPanel we are switching out to different components using SectionList when switching tabs, and the delayed mounting here caused redrawing everything. It isn't needed anymore now because we're adding a delay here now https://github.com/keybase/client/pull/23403/files#diff-e248652887262a0acc494ea036f08e19R191

Reply:
Where's the delay we're adding now?

Reply:
huh weird the link doesn't work. Line 190 of this file